pontoon/base/urls.py

```diff
@@ -30,6 +30,15 @@
     url(r'^contributor/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
         RedirectView.as_view(url="/contributors/%(email)s/", permanent=True)),
 
+    # List all active localization teams
+    url(r'^teams/$',
+        views.locales,
+        name='pontoon.teams'),
+
+    # Redirect to a team page
+    url(r'^teams/(?P<locale>[A-Za-z0-9\-\@\.]+)/$',
```

Comment:
Also, I think perhaps `/locale/sl` should be the canonical URL rather than just `/sl`. As you already noticed, having a weird catch-all URL makes routing more complicated (hence why it's the last route in the urlconf). But we also probably don't want to break the old links if a bunch of external things are linking to them; do you have a sense of if anything is linking to pages like https://pontoon.mozilla.org/sl currently?